### PR TITLE
chain: Relayer test

### PR DESCRIPTION
**Pull request by perimeko**
Fixed: 1692

### CODE REVIEW

File: chain/x/oracle/keeper/relayer_test.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+	seq := uint64(1)
+
+	testCases := []struct {
+		description      string
+		exec             func(log.Logger) (*app.BandApp, sdk.Context)
+		expectedPass     bool
+		expectedErrorLog string
+	}{
+		{
+			"success send IBC packet",
```
Comment (sorawit) [inline chain/x/oracle/keeper/relayer_test.go]: successfully

File: chain/x/oracle/keeper/relayer_test.go [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+	header := ibctmtypes.CreateTestHeader(chain.Name(), 1, now, valSet, signers)
+
+	return chain.NewContext(false, abci.Header{
+		ChainID: header.ChainID,
+		Height:  header.Height,
+		Time:    now,
+	})
+}
+
+func createTestClient(chainA *bandapp.BandApp, chainB *bandapp.BandApp) error {
```
Comment (sorawit) [inline chain/x/oracle/keeper/relayer_test.go]: Let's just panic on error so this function doesn't need to return anything

File: chain/x/oracle/keeper/relayer_test.go [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+		ctx = ctx.WithBlockHeight(2)
+
+		packet := types.OracleResponsePacketData{
+			ClientID:      "alice",
+			RequestID:     3,
+			AnsCount:      1,
+			RequestTime:   1589535020,
+			ResolveTime:   1589535022,
+			ResolveStatus: 1,
+			Result:        []byte("4bb10e0000000000"),
```
Comment (sorawit) [inline chain/x/oracle/keeper/relayer_test.go]: Let's put arbitrary string here

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/x/oracle/keeper/relayer_test.go [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+	TestPortA                                           = "testporta"
+	TestPortB                                           = "testportb"
+	TestChannelA                                        = "testchannela"
+	TestChannelB                                        = "testchannelb"
+	TestConnectionA                                     = "connectionAtoB"
+	TestConnectionB                                     = "connectionBtoA"
+	TrustingPeriod                        time.Duration = time.Hour * 24 * 7 * 2
+	UbdPeriod                             time.Duration = time.Hour * 24 * 7 * 3
+	MaxClockDrift                         time.Duration = time.Second * 10
+	DefaultPacketTimeoutHeight                          = 0
```
Comment (sorawit) [inline chain/x/oracle/keeper/relayer_test.go]: Let's reuse the values from keeper

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**